File: boot-tyb/src/main/java/com/xzm/modules/tyb/pojo/TybUser.java
package com.xzm.modules.tyb.pojo;

import lombok.Data;

import java.util.Date;
@Data
public class TybUser {
    private Integer id;

    private String nickName;

    private String phone;

    private String password;

    private String phoneUrl;

    private Integer pushStatus;

    private Date createTime;

    private Date updataTime;


}

File: boot-tyb/src/main/java/com/xzm/tyb/pojo/entity/TybUser.kt
package com.xzm.tyb.pojo.entity

import com.xzm.tyb.utils.identity
import java.util.Date

//@Data
class TybUser {
    var id: Long? = 0
//    var id: Long = identity.nextId()
    var nickName: String? = null

    var phone: String? = null

    var password: String? = null

    var phoneUrl: String? = null

    var pushStatus: Int? = null

    var createTime: Date? = null

    var updataTime: Date? = null

}